core/templates/pages/contributor-dashboard-page/opportunities-list-item/opportunities-list-item.component.spec.ts

```diff
@@ -20,13 +20,26 @@ import { ComponentFixture, fakeAsync, TestBed } from '@angular/core/testing';
 import { LazyLoadingComponent } from 'components/common-layout-directives/common-elements/lazy-loading.component';
 import { NgbTooltipModule } from '@ng-bootstrap/ng-bootstrap';
 import { WrapTextWithEllipsisPipe } from 'filters/string-utility-filters/wrap-text-with-ellipsis.pipe';
+import { of } from 'rxjs';
+import { WindowDimensionsService } from 'services/contextual/window-dimensions.service';
 
 import { ExplorationOpportunity, OpportunitiesListItemComponent } from './opportunities-list-item.component';
 import { ContributorDashboardConstants } from 'pages/contributor-dashboard-page/contributor-dashboard-page.constants';
 
+class MockWindowDimensionsService {
+  getResizeEvent() {
+    return of(new Event('resize'));
+  }
+
+  getWidth(): number {
+    return 530;
```

Comment:
Where does this number come from? Would be good to document in a comment.

Reply:
This can be any arbitrary number below 700 to simulate a mobile viewport. I have changed it to 390 (width for iPhone 12) to be a bit more specific.